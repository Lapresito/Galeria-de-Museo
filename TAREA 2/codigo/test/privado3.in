crearColeccionPiezas
crearFecha 2/2/2002
crearPieza 10 LaxaEspadaTriste Sapphire Frost
agregarColeccionPiezas
crearFecha 4/3/2005
crearPieza 30 SacraLataFurtiva River Rain
agregarColeccionPiezas
crearFecha 2/2/2006
crearPieza 50 GozadaEspadaDesquiciada River Rain
agregarColeccionPiezas
crearFecha 6/5/2007
crearPieza 70 RaraMangueraDesquiciada River Rain
agregarColeccionPiezas
crearFecha 6/5/2008
crearPieza 40 MalditaLataFunebre River Rain
agregarColeccionPiezas
crearFecha 6/5/2009
crearPieza 60 AltaBarcaAfilada River Rain
agregarColeccionPiezas
removerDeColeccionPiezas 50
removerDeColeccionPiezas 10
esVaciaColeccionPiezas
crearFecha 2/2/2006
crearPieza 50 GozadaEspadaDesquiciada River Rain
agregarColeccionPiezas
obtenerPiezaColeccionPiezas 50
obtenerPiezaColeccionPiezas 40
obtenerPiezaColeccionPiezas 60
esVaciaColeccionPiezas
imprimirColeccionPiezas
liberarColeccionPiezas
Fin
