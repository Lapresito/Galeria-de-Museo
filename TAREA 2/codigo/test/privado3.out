Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> Coleccion de piezas creada de forma exitosa.
2> Fecha creada en forma exitosa.
3> Pieza creada en forma exitosa.
4> Pieza agregada de forma exitosa.
5> Fecha creada en forma exitosa.
6> Pieza creada en forma exitosa.
7> Pieza agregada de forma exitosa.
8> Fecha creada en forma exitosa.
9> Pieza creada en forma exitosa.
10> Pieza agregada de forma exitosa.
11> Fecha creada en forma exitosa.
12> Pieza creada en forma exitosa.
13> Pieza agregada de forma exitosa.
14> Fecha creada en forma exitosa.
15> Pieza creada en forma exitosa.
16> Pieza agregada de forma exitosa.
17> Fecha creada en forma exitosa.
18> Pieza creada en forma exitosa.
19> Pieza agregada de forma exitosa.
20> Pieza con id 50 removida con exito.
21> Pieza con id 10 removida con exito.
22> La coleccion de piezas NO es vacia.
23> Fecha creada en forma exitosa.
24> Pieza creada en forma exitosa.
25> Pieza agregada de forma exitosa.
26> Pieza: 50
GozadaEspadaDesquiciada-River Rain
Creada en: 2/2/2006
27> Pieza: 40
MalditaLataFunebre-River Rain
Creada en: 6/5/2008
28> Pieza: 60
AltaBarcaAfilada-River Rain
Creada en: 6/5/2009
29> La coleccion de piezas NO es vacia.
30> Pieza: 30
SacraLataFurtiva-River Rain
Creada en: 4/3/2005
Pieza: 40
MalditaLataFunebre-River Rain
Creada en: 6/5/2008
Pieza: 50
GozadaEspadaDesquiciada-River Rain
Creada en: 2/2/2006
Pieza: 60
AltaBarcaAfilada-River Rain
Creada en: 6/5/2009
Pieza: 70
RaraMangueraDesquiciada-River Rain
Creada en: 6/5/2007
31> Coleccion liberada con exito.
32> Fin.
